arcgis-ios-sdk-samples/Display information/Create symbol styles from web styles/CreateSymbolStylesFromWebStylesViewController.swift

```diff
@@ -56,28 +56,27 @@ class CreateSymbolStylesFromWebStylesViewController: UIViewController, UIAdaptiv
     
     // MARK: Methods
     
-    /// Create a map with an `AGSFeatureLayer` added to its operational layers.
-    /// - Parameter featureLayer: An `AGSFeatureLayer` object.
-    /// - Returns: An `AGSMap` object.
-    func makeMap(featureLayer: AGSFeatureLayer) -> AGSMap {
+    /// Create a map with a reference scale.
+    func makeMap(referenceScale: Double) -> AGSMap {
         let map = AGSMap(basemapStyle: .arcGISLightGray)
-        map.referenceScale = 1e5
-        map.operationalLayers.add(featureLayer)
+        map.referenceScale = referenceScale
         return map
     }
     
     /// Create an `AGSUniqueValueRenderer` to render feature layer with symbol styles.
-    /// - Parameter fieldNames: The attributes to match the unique values against.
+    /// - Parameters:
+    ///   - fieldNames: The attributes to match the unique values against.
+    ///   - symbolInfos: A dictionary of symbols and their associated infos.
     /// - Returns: An `AGSUniqueValueRenderer` object.
-    func makeUniqueValueRenderer(fieldNames: [String]) -> AGSUniqueValueRenderer {
+    func makeUniqueValueRenderer(fieldNames: [String], symbolInfos: [AGSSymbol: (name: String, values: [String])]) -> AGSUniqueValueRenderer {
```

Comment:
Passed an argument to the method of a dict of symbols and infos. A few reasonings

- Instead of directly using the element-wise initializer of `AGSUniqueValueRenderer` in the `DispatchGroup.notify` block, I created this method because I think it is important to separately showcase how we map the symbols' names and category values to create unique values.
- To avoid large tuples, I used dictionary to pass those computed strings
- Naming: I'm thinking either `symbolAttributes` or `symbolInfos` and chose the latter.

though I'm still not quite satisfied. Love to hear more feedback!

Reply:
I think what you have here is fine. Another name idea is `symbolDetails`.